CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# v0.0.7
+* Address review comments.
+  Humble Chirammal <hchiramm@redhat.com>
+
```

Comment:
The changelog version and entries are matching with the docker image tags we use for subject containers, so it make sense.  One additional thing I can do here is, update the version string in the code as soon as there is a changelog entry.

Reply:
I agree that since we don't have any releases, we shouldn't have them here in the changelog w/ versions. There definitely shouldn't be tagged image versions w/o a github release.

I think this file can be collapsed to:
```
# master
- Intital version of CSI driver
```

When we cut a release, we can change `master` to the releaase tag